docs/standard/data/xml/managing-namespaces-in-an-xml-document.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Managing Namespaces in an XML Document"
+description: Learn how to manage namespaces in an XML document. XML namespaces associate element and attribute names in an XML document with custom and predefined URIs.
```

Comment:
I just noticed a few of these use a second sentence that repeats the first paragraph in the article. Is that valid for SEO? I thought we're not supposed to repeat?

Reply:
From the guidance (https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=master):

_Use text from the introductory paragraph of on-page copy edited for length, if it describes the article intent._